_data/languages.yml

```diff
@@ -0,0 +1,6 @@
+- language:
+  code: en
```

Comment:
I don't know a lot of yaml but I think I read that the "language:" part should not be needed. You can directly add the dash before the code and remove the empty "language" key

```suggestion
- code: en
```